cmd/amplifier/server/configuration.go

```diff
@@ -10,17 +10,17 @@ import (
 )
 
 const (
-	DefaultPort          = ":50101"
-	DefaultPublicAddress = "local.appcelerator.io"
-	DefaultTimeout       = time.Minute
+	DefaultPort       = ":50101"
+	DefaultPublicHost = "127.0.0.1"
```

Comment:
we may have an issue once we're ready to do TLS termination outside of amplifier, if we use a different port, the exposed address won't match